app/facades/foods_facade.rb

```diff
@@ -0,0 +1,17 @@
+class FoodsFacade
+  attr_reader :search_query
+
+  def initialize(food_info)
+    @search_query = food_info[:q]
+  end
+
+  def food_search
+    FoodDataService.new.food_search(@search_query)[:foods].first(10).map do |food|
```

Comment:
This is good, but I think we can be more efficient by asking for the `.first(10)` either from the API service itself (if it allows it, e.g. `&pageSize=10`) or in the Service instead. That way, all that loads of data doesn't get passed to the facade and instead it only gets the total number it needs to process.